views/services.jade

```diff
@@ -10,7 +10,7 @@ block content
 
       if service.image
         .section-wrapper.image.th
-          img(src="/images/content/services/#{service.image}", alt= pageTitle, itemprop="image")
+          img(src="/images/content/services/lake.jpg", alt= pageTitle, itemprop="image")
```

Comment:
By making this change, all services pages are using the same lake image. This should be pulled dynamically from the database, so it should be left unchanged. Your update in `services.json` will cover it. Just `make reset` to pull it in to your DB for testing.